.github/workflows/opt.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
         if: github.event_name == 'workflow_dispatch' && inputs.pr_number != ''
         uses: thollander/actions-comment-pull-request@v3
         with:
-          pr_number: ${{ inputs.pr_number }}
+          pr-number: ${{ inputs.pr_number }}
```

Comment:
Just so I don't forget: this is also fixed upstream, might just have been an issue with the `git` histories